### DISCUSSION TITLE
I have been kept in the waiting list the fourth week now.

### DISCUSSION THREAD

**Discussion by charity1475** [community/community#8165] [Copilot Conversations] [answered]
My friend got it in the first week but : charitymbisi@gmail.com , taking the fourth week the next week is the fifth one.

**Comment by jonaslsaa** [comment]
I had to wait a few months, be patient.

**Comment by davecheney** [comment, marked as answer]
We are adding new invites as we have capacity. Thank you for your patience and understanding.
